### DISCUSSION TITLE
"Open with" functionality support for editing a file using a flet application.

### DISCUSSION THREAD

**Discussion by manupatel007** [flet-dev/flet#3273] [Q&A]
### Question

Is it possible to have "Open with" functionality support for editing a file using a flet application?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Can you be more detailed and specific?

**Reply by manupatel007** [reply]
Sure, when we open a text file in windows, we have an option of "open with" suggesting whether we want to open it with Notepad, Wordpad, VS code etc. as shown below

![Screenshot (1537)](https://github.com/flet-dev/flet/assets/52189866/08a7b50b-5fdf-4b9e-b47e-8dc053870589)

I want to understand if something similar can be achieved in flet. I know that we can utilize file upload functionality/read file from path and load it in flet application. But just wanted to know if this particular this is possible. Thanks.

**Reply by ndonkoHenri** [reply]
No idea.
You can try searching the internet on how that is done in flutter, then share it here for possible implementation.
